.github/workflows/ci.yml

```diff
@@ -41,6 +41,16 @@ jobs:
           sudo tlmgr install standalone preview doublestroke relsize fundus-calligra wasysym physics dvisvgm.x86_64-darwin dvisvgm rsfs wasy cm-super
           echo "::set-env name=PATH::$PATH"
 
+      - name: Install system dependencies (Windows)
+        if: runner.os == 'Windows'
+        run: |
+          Invoke-WebRequest https://yihui.org/gh/tinytex/tools/install-windows.bat -O install_tinytex.bat
+          Start-Process install_tinytex.bat -NoNewWindow -Wait
+          $env:Path += ";" + $env:APPDATA + "\TinyTeX\bin\win32"
```

Comment:
If you use the `::set-env` workflow command you shouldn't have to repeat this when running `pytest`.

https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Apparently if I add to the path via that, I can't access it in the same 'action' that created it in the first place, at least, that's what [the link you cited](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path) says.

Reply:
If you add the command at the end of the action similarly to how it is for the MacOS setup then the updated PATH should be carried over to the pytest step. Then you wouldn't have to repeat it and Windows can use the same pytest action as the other OSes.

Reply:
Ah got it. It's done in the latest commit.